kubeflow/optimizer/backends/kubernetes/backend.py

```diff
@@ -290,6 +291,11 @@ def wait_for_job_status(
                 f"{constants.OPTIMIZATION_JOB_KIND} {name}, status {optimization_job.status}"
             )
 
+            # Invoke callbacks if provided
+            if callbacks:
+                for callback in callbacks:
+                    callback(optimization_job)
```

Comment:
Would that make sense to enable the callback to return a `Optional[bool]` that would control whether the polling should stop?

Reply:
Yeah, but the `wait_for_job_status()` returns object only when it reaches the desired status.
As I mentioned, some users would like to periodically print job logs, events, and statuses while waiting for the desired condition: `Complete`.